### ISSUE TITLE
When training one's own dataset, if the learning rate is increased but the loss still cannot be reduced, is there any solution?

### ISSUE THREAD

**Issue by Meantao** [RT-DETRs/RT-DETRv4#16]
This is a very good job. However, when I was training my dataset and increased the learning rate, the loss still couldn't be reduced and the metrics weren't as high as those in the paper. Is there any solution?

**Comment by Zhao-Yian** [comment]
Thank you for your attention. 

1. Ensure the dataset annotation format is correct (COCO format); 
2. Try loading the pre-trained weights.

**Comment by Meantao** [comment]
Do you mean that the pre-trained model refers to dinov3？

**Comment by Zhao-Yian** [comment]
Loading the COCO-pretrained RT-DETRv4 model.

**Comment by Meantao** [comment]
I have already load the COCO-pretrained RT-DETRv4 model, But still get low mAP in our dataset.

**Comment by nikky4D** [comment]
@Meantao did you resolve your issue?
